agent/host_agent_test.go

```diff
@@ -542,7 +542,7 @@ var _ = Describe("Agent", func() {
 
 			runner = setupTestInfra(ctx, hostName, getKubeConfig().Name(), ns)
 			runner.CommandArgs["--feature-gates"] = "SecureAccess=true"
-
+			runner.CommandArgs["--bootstrap-kubeconfig"] = "/mgmt.conf"
```

Comment:
Actually wanted to use the same `kubeconfig` for the bootstrap process as well. Otherwise, have to create another config, which could be handled in `e2e` case. WDYT?

Reply:
> same kubeconfig for the bootstrap process as well

Didn't get this. What bootstrap process are you referring to?

Reply:
I mean we cannot rename `mgmt.conf` to `bootstrap.conf` or otherwise we have to create another kubeconfig. I guess we can make these refactor in the final changes to support only one flow.

Reply:
Let's continue this discussion for the next developments.